src/carousel/carousel.scss

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../text/index';
+
+@mixin iui-carousel {
+  @include iui-reset;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+@mixin iui-carousel-slider {
+  position: relative;
```

Comment:
Any reason you remember for adding this?

Reply:
I removed it and it doesn't seem to be changing anything, so it is gone.